### PR TITLE
Meta: Automatically generate release notes again /2

### PR DESCRIPTION
**Pull request by FloEdelmann**
Follow-up to #5223, this time with the correct step name (see https://github.com/refined-github/refined-github/pull/5223#issuecomment-1000299508).

~This is still untested.~ Tested now: https://github.com/refined-github/refined-github/pull/5243#issuecomment-1003366051

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -30,6 +30,16 @@ jobs:
...
         name: Create tag if necessary
         id: daily-version
+      - if: steps.daily-version.outputs.created
+        name: Create release
+        uses: actions/github-script@v5.0.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            await github.request(`POST /repos/${{ github.repository }}/releases`, {
+              tag_name: "${{ steps.daily-version.outputs.version }}",
```
Comment (fregante) [inline .github/workflows/release.yml:40]: eeeeee that might explain the error 😒 👍

**Comment by fregante** [comment]
I think you can actually test this on your repo by running it manually. We only need to check the Version step. You just need to [enable actions in the settings](https://github.com/FloEdelmann/refined-github/settings/actions) and then go to 

- https://github.com/FloEdelmann/refined-github/actions/workflows/release.yml

Screen Shot 5

**Comment by FloEdelmann** [comment]
Done.

Workflow: https://github.com/FloEdelmann/refined-github/runs/4673942230
Created release: https://github.com/FloEdelmann/refined-github/releases/tag/21.12.31

**Comment by fregante** [comment]
It worked! https://github.com/refined-github/refined-github/releases/tag/22.1.11

**Pull request merged**